### PR TITLE
RStudio 1.10.0: Bumped RStudio to version 1.2

### PR DESCRIPTION
**Pull request by xoen**
RStudio release notes: https://blog.rstudio.com/2019/04/30/rstudio-1-2-release/
Ticket: https://trello.com/c/rguKYALc/214-try-to-upgrade-rstudio-docker-image-to-use-latest-version

### CODE REVIEW

**Comment by xoen** [comment]
Update: I tried to base the RStudio image off `rocker/verse:3.6.0` but I got the same problem so moving on. I'll leave this open for now as I may want to give it a try in few days/weeks hoping they fix that problem.

**Comment by xoen** [comment]
Nope. Used that base image didn't work neither so we'll have to wait for RStudio to fix this connection error problem upstream.

File: charts/rstudio/values.yaml [hunk 1]
```diff
@@ -30,7 +30,7 @@ rstudio:
   port: "8787"
   image:
     repository: quay.io/mojanalytics/rstudio
-    tag: "r3.5.1-py3.7-conda-1"
+    tag: "1.2.1335-r3.5.1-python3.7.1-conda-3"
```
Comment (xoen) [inline charts/rstudio/values.yaml]: @r4vi as discussed I've bumped the patch bit but it's worth double checking this.

I'll obviously hold fire on this till the release/tag is created.

**Comment by r4vi** [comment]
@xoen I think @RobinL should be the first one to try this :laughing:

**Comment by xoen** [comment]
"first" after Thomas H. 😋

**Comment by r4vi** [comment]
@xoen builds have finished feel free to merge

**Pull request merged**